local-volume/provisioner/cmd/main.go

```diff
@@ -70,13 +75,16 @@ func getNode(client *kubernetes.Clientset, name string) *v1.Node {
 	return node
 }
 
-func createDiscoveryMap() map[string]common.MountConfig {
-	// Default setting
-	// TODO: change this to configurable settings.
-	m := make(map[string]common.MountConfig)
-	m["local-storage"] = common.MountConfig{
-		HostDir:  "/mnt/disks",
-		MountDir: "/local-disks",
+func createDiscoveryMap(client *kubernetes.Clientset) map[string]common.MountConfig {
```

Comment:
Can you elaborate the request?  I logged the config per previous request, so it should be sufficient.

Or do you mean we should log config passed from user (instead of default one?)